Automations/updateFrontend.sh

```
@@ -0,0 +1,31 @@
+YELLOW='\033[0;33m'
+NC='\033[0m'
+
+# Use curl to fetch the public IPv4 address from the metadata service
+ipv4_address=$(curl -s http://169.254.169.254/latest/meta-data/public-ipv4)
+
+echo -e " ${GREEN}System Public Ipv4 address ${NC} : ${ipv4_address}"
+
+if [[ "${alreadyUpdate}" == "VITE_API_PATH=\"http://${ipv4_address}:31100\"" ]]
+then
+        echo -e "${YELLOW}${file_to_find} file is already updated to the current host's Ipv4 ${NC}"
+	exit -1;
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)